app/components/Mods/AlphaSwitch.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Switch, Tooltip, FormLabel } from '@material-ui/core';
+import { useSetRecoilState, useRecoilValue } from 'recoil';
+import { alphaState } from '../../store';
+
+const AlphaSwitch: React.FunctionComponent = () => {
+    const setAlpha = useSetRecoilState(alphaState);
+    const alpha = useRecoilValue(alphaState);
+    return (
+        <div>
+            <Tooltip title={'View the mods created for Outer Wilds Alpha.'}>
```

Comment:
I know I just said to remove this switch, but remember to have all translatable text in `static-text.ts`